tests/python/gaia-ui-tests/gaiatest/apps/messages/app.py

```diff
@@ -11,6 +11,7 @@
 class Messages(Base):
 
     name = 'Messages'
+    manifest_url = "app://sms.gaiamobile.org/manifest.webapp"
```

Comment:
So I understand Right it should be like that: 
manifest_url = format(self.DEFAULT_PROTOCOL).'sms/'.format(self.DEFAULT_APP_HOSTNAME).'/manifest.webapp'